.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
```

Comment:
We can remove this file and add this file type to `.gitignore`.

Reply:
okay will add it to .gitignore